region/maridia/inner-green.json

```diff
@@ -132,57 +166,131 @@
           "to": [
             {
               "id": 1,
-              "requires": null
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": null
+                }
+              ]
             },
             {
               "id": 2,
-              "requires": null
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": null
+                }
+              ]
             },
             {
               "id": 4,
-              "requires": [
-                {"or": [
-                  {"canComeInCharged": {
-                    "fromNode": 1,
-                    "framesRemaining": 30,
-                    "shinesparkFrames": 25
-                  }},
-                  {"canComeInCharged": {
-                    "fromNode": 2,
-                    "framesRemaining": 30,
-                    "shinesparkFrames": 25
-                  }},
-                  {"and": [
-                    "Gravity",
+              "strats": [
+                {
+                  "name": "Come in Charged",
+                  "notable": false,
+                  "requires": [
                     {"or": [
-                      "canDestroyBombWalls",
-                      {"and": [
-                        "canXRayClimb",
-                        {"resetRoom":{
-                          "nodes": [1],
-                          "mustStayPut": true
-                        }}
-                      ]}
+                      {"canComeInCharged": {
+                        "fromNode": 1,
+                        "framesRemaining": 30,
+                        "shinesparkFrames": 23
+                      }},
+                      {"canComeInCharged": {
+                        "fromNode": 2,
+                        "framesRemaining": 30,
+                        "shinesparkFrames": 23
+                      }}
                     ]}
-                  ]},
-                  {"and": [
+                  ],
+                  "note": [
+                    "Not sure whether this should require canNavigateUnderwater?",
```

Comment:
It does make use of water physics to reach the top platform 🤷‍♀

Reply:
Between that and the fact that positioning the spark can be finicky depending on incoming speed, I'll just require it.